src/pybind/mgr/dashboard_v2/frontend/src/app/ceph/block/pool-detail/pool-detail.component.ts

```diff
@@ -0,0 +1,93 @@
+import { Component, OnDestroy, OnInit } from '@angular/core';
+import { ActivatedRoute } from '@angular/router';
+
+import { ViewCacheStatus } from '../../../shared/enum/view-cache-status.enum';
+import { DimlessBinaryPipe } from '../../../shared/pipes/dimless-binary.pipe';
+import { DimlessPipe } from '../../../shared/pipes/dimless.pipe';
+import { FormatterService } from '../../../shared/services/formatter.service';
+import { PoolService } from '../../../shared/services/pool.service';
+
+@Component({
+  selector: 'cd-pool-detail',
+  templateUrl: './pool-detail.component.html',
+  styleUrls: ['./pool-detail.component.scss']
+})
+export class PoolDetailComponent implements OnInit, OnDestroy {
+
+  name: string;
+  images: any;
+  columns: any;
+  retries: number;
+  maxRetries = 5;
+  routeParamsSubscribe: any;
+
+  constructor(private route: ActivatedRoute,
+              private poolService: PoolService) {
+    this.columns = [
+      {
+        name: 'Name',
+        prop: 'name',
+        width: 100
+      },
+      {
+        name: 'Size',
+        prop: 'size',
+        width: 50,
+        cellClass: 'text-right',
+        pipe: new DimlessBinaryPipe(new FormatterService())
```

Comment:
If you inject this Pipe via the constructor you can then just define `pipe: this.dimlessBinary`.

Reply:
Nice tip! I've adapter the code as you suggested.